app/_includes/md/ce-kong-user.md

```diff
@@ -0,0 +1,9 @@
+<!-- Shared between all Community Linux installation topics: Amazon Linux,
+ CentOS, Debian, RedHat, and Ubuntu -->
+
+    **Note:** When you start Kong, by default, the Nginx master process will run
+    as `root` and the worker processes will run as `nobody`.
```

Comment:
```suggestion
    as `root` and the worker processes run as `nobody` by default.
```

otherwise, it sounds awkward "when you start Kong, by default"